Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3585,6 +3588,40 @@ extension BrowserViewController: UNUserNotificationCenterDelegate {
   }
 }
 
+// MARK: UIScreenshotServiceDelegate
+
+extension BrowserViewController: UIScreenshotServiceDelegate {
+  
+  @MainActor
+  public func screenshotServiceGeneratePDFRepresentation(_ screenshotService: UIScreenshotService) async -> (Data?, Int, CGRect) {
+    await withCheckedContinuation { continuation in
```

Comment:
If you're going to use async/await here then dump the checked continuation entirely and use async version of `createPDF`?

Reply:
Turns out WebKit defines the async pdf creation method as 15+… we can revisit after dropping 14